Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -31,32 +30,50 @@ struct SwiftUINativeCheckoutDemo: View {
     }
 
     func checkoutView(_ title: String, _ content: String, _ isFlowComplete: Bool) -> some View {
-        NavigationView {
-            ZStack {
-                VStack(spacing: 16) {
-                    Picker("Checkout ", selection: $checkoutTypeSelection) {
-                        ForEach(CheckoutType.allCases) {type in
-                            Text(type.rawValue)
-                        }
+        VStack(spacing: 16) {
+            HStack {
+                Text("Checkout:")
```

Comment:
Maybe since this is a picker we can update this text to something like "Checkout type selection" or something to indicate that there are multiple options. May also be able to do a list picker like in the apple docs example (though not sure if you'd run into the same issues as a form): https://developer.apple.com/documentation/swiftui/picker

Reply:
I tried with the list picker and it has the same problems as with Form, it just takes to much space and i have to do some hacky stuff to reduce it.

Reply:
That makes sense! 👍